lib/servers/iXonServer/AndorVideo.py

```diff
@@ -1,4 +1,5 @@
-from config.andor_config import andor_config as config
+import andor_config as config
```

Comment:
the old format sounds great to me.  I though it changed here based on the .gitignore.